chart/epinio/templates/service-catalog.yaml

```diff
@@ -71,7 +71,7 @@ spec:
     You can find more info at https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq/.
     This instance is running inside the cluster so it's probably not a good choice for production
     environments, at least with this default configuration.
-  chart: rabbitmq
+  chart: rabbitmq:9.0.3
```

Comment:
This is already outdated! They released the `9.0.4` just today: [https://artifacthub.io/packages/helm/bitnami/rabbitmq](https://artifacthub.io/packages/helm/bitnami/rabbitmq)